Tests/regressionTests.js

```diff
@@ -0,0 +1,301 @@
+        .then(() => loginpage.enterEmail(userName))
+        .then(() => loginpage.enterPassword(password))
+        .then(() => loginpage.clickOnLoginButton())
+        .then(() => browser.wait(EC.presenceOf($(".have-account")), 5000))
+        .then(() => loginpage.getLoginURL())
+        .then((data) => expect(data).toBe("https://auct-app2-frontend.herokuapp.com/register"))
+    }) 
+    it ("004: User isn't able to log in without password", function(){
+        var userName = "rafa@mail.com"
+        homepage.clickOnLogin()
+        .then(() => loginpage.enterEmail(userName))
+        .then(() => loginpage.clickOnLoginButton())
+        .then(() => browser.sleep(2000))
+        .then(() => loginpage.getLoginValidationMessage())
+        .then((data) => expect(data).toBe("Invalid username or password"))
+    })
+    it("005: User isn't able to log in without email", function(){
+        var password= "password";
+        homepage.clickOnLogin()
+        .then(() => loginpage.enterPassword(password))
+        .then(() => loginpage.clickOnLoginButton())
+        .then(() => browser.sleep(2000))
+        .then(() => loginpage.getLoginValidationMessage())
+        .then((data) => expect(data).toBe("Invalid username or password"))
+    })
+})
+describe("004: Bidding", function(){
+    beforeEach(()=> {
+        homepage.open("https://auct-app2-frontend.herokuapp.com/")
+        .then(() => homepage.clickOnLogin())
+        .then(() => loginpage.enterEmail("rafa@mail.com"))
+        .then(() => loginpage.enterPassword("password"))
+        .then(() => loginpage.clickOnLoginButton())
+        .then(() => browser.wait(EC.presenceOf($(".category-item")), 5000))
+    })
+    afterEach(() =>{
+        homepage.clickOnLogoutButton();
+    })
+    it ("001: User is able to make a bid with decimal values", function (){
+        var d = new Date();
+        var months = ["January", "February", "March", "April", "May", "June", "July", "August", "September", "October", "November", "December"];
+        homepage.clickOnFashionCategory()
+        .then(() => browser.wait(EC.presenceOf($(".landing-product")), 5000))
+        .then(() => fashionCategory.clickOnBrownJacket())
+        .then(() => itemPage.enterBid(400.25))
+        .then(() => itemPage.clickOnPlaceBidButton())
+        .then(() => browser.wait(EC.presenceOf($(".bider-name-img")), 5000))
+        .then(() => itemPage.getUserName())
+        .then((data) => expect(data).toBe("Rafa Nadal"))
+        .then(() => itemPage.getBidPrice())
+        .then((data) => expect(data).toBe("$400.25.00"))
+        .then(() => itemPage.getBidDate())
+        .then((data) => expect(data).toBe(`${d.getDate()} ${months[d.getMonth()]} ${d.getFullYear()}`))
+    });
+    it("002: User is able to make a bid with round values", function(){
+        var d = new Date();
+        var months = ["January", "February", "March", "April", "May", "June", "July", "August", "September", "October", "November", "December"];
```

Comment:
all used data should be removed to separated class

Reply:
I created all data to Data class